Vagrant-bootstrap.sh

```diff
@@ -21,7 +21,8 @@ function install_build_dependencies(){
 }
 
 ### apt update, install nice-to-have packages and build dependencies
-
+## change mirror, uncomment following and change mirror according to your location
+# sed -i 's/deb.debian.org/mirrors.ustc.edu.cn/g' /etc/apt/sources.list
 apt-get update
```

Comment:
Hi @xgdgsc , thank you very much for raising this PR, I really appreciate it.

I was wondering if you would be open to moving this block up a bit and also adding a link to the list of official Debian mirrors:

```suggestion

### use a specific mirror
# a complete list of Debian mirrors can be found on this page: https://www.debian.org/mirror/list#complete-list
# uncomment the line below and replace the example mirror (foo.bar.org) according to your location
# sed -i 's/deb.debian.org/foo.bar.org/g' /etc/apt/sources.list

### apt update, install nice-to-have packages and build dependencies

apt-get update
```